src/components/apps/launch/AppLaunchForm.js

```diff
@@ -64,35 +64,48 @@ import {
     useTheme,
 } from "@material-ui/core";
 
-import { ArrowBack, ArrowForward, PlayArrow, Save } from "@material-ui/icons";
+import {
+    ArrowBack,
+    ArrowForward,
+    KeyboardArrowLeft,
+    KeyboardArrowRight,
+    PlayArrow,
+    Save,
+} from "@material-ui/icons";
 
 const useStyles = makeStyles(styles);
 
 // sonora appbar height.
 //Stepper height, appinfo height and bottom nav height is calculated dynamically.
 const NAVIGATION_BAR_HEIGHT = 105;
-const MOBILE_NAVIGATION_BAR_HEIGHT = 75;
+
+//include mobile stepper navigation height
+const MOBILE_NAVIGATION_BAR_HEIGHT = 135;
 
 const ReferenceGenomeParamTypes = [
     constants.PARAM_TYPE.REFERENCE_GENOME,
     constants.PARAM_TYPE.REFERENCE_SEQUENCE,
     constants.PARAM_TYPE.REFERENCE_ANNOTATION,
 ];
 
-const StepContent = ({ id, hidden, step, label, children, offsetHeight }) => (
-    <PageWrapper appBarHeight={offsetHeight}>
-        <fieldset id={id} hidden={hidden}>
-            <legend>
-                <Typography variant="caption">
-                    {getMessage("stepLabel", {
-                        values: { step: step + 1, label },
-                    })}
-                </Typography>
-            </legend>
-            {children}
-        </fieldset>
-    </PageWrapper>
-);
+const StepContent = ({ id, hidden, step, label, children, offsetHeight }) => {
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
+    return (
+        <PageWrapper appBarHeight={offsetHeight}>
+            <fieldset id={id} hidden={hidden}>
+                <legend>
+                    <Typography variant={isMobile? "subtitle2": "caption"} color={isMobile ? "primary" : "inherit"}>
```

Comment:
Since mobile stepper doesn't have separate step identification UI, I made the step # and name little prominent.